README.md

```diff
@@ -1,15 +1,76 @@
-# Fast Store
+# FastStore 
+## An open ecommerce framework
 
-https://docs.google.com/presentation/d/10SCY7ZO1WWGeUQs6VzJWAlp4r1404dVtIxx_YgYldZI/edit#slide=id.g8ef796eccd_0_336
+FastStore is an open ecommerce framework built using modern technologies that aims to deliver the building blocks necessary for creating your custom ecommerce experience. With these building blocks you can archieve:
 
-https://miro.com/app/board/o9J_kobcu-Y=/
-This is the home for the proof of concept project to create VTEX sites based on the Gatsby framework.
+- Multiregion stores with native internationalization support
+- Great Performance (90+ on Google's page speed insights)
+- SEO ready pages with Google's Rich Results support
+- CMS managed content
+- Jamstack based (deploy with Netlify, Vercel and any other JAMStack platform)
+- Marketplace ready with support to millions of skus
+- Intelligence via Google Analytics and GTM
+- Evergreen stores with our no-fork based approach
 
-# How to develop
+## Why building FastStore?
+The web is a wild and noisy environment. Many projects claims to solve many different aspects of web development but none of them solves the ecommerce problem satisfactory. Ecommerce is a very complex case where performance is a must, but personalization is the key. 
+
+After years of developing ecommerces, FastStore creators packaged all of this knowledge into easy, ready to use building blocks for crafting production ready, multi-region marketplaces.
+
+FastStore not only contains code, but knowledge and guidelines to change the way you think and build ecommerces
+
+## Project philosophy
+This project has a very well defined list of priorities. When solving an issue, a lower ranking priority must not degrade or override an upper ranking priority.
+The priorities are:
+
+1. Performance
```

Comment:
Performance — we applied a server-side rending logic to load pages faster and made monitoring easer with our CI integration

Reply:
Not only SSR right ? Also shrinking the DOM, layout reflow, and static analysis